app/models/concerns/orchestration.rb

```diff
@@ -44,9 +44,13 @@ def failure(msg, backtrace = nil, dest = :base)
   # after validation callbacks status, as rails by default does
   # not care about their return status.
   def valid?(context = nil)
-    setup_clone
-    super
-    orchestration_errors?
+    if SETTINGS[:unattended]
+      setup_clone
+      super
+      orchestration_errors?
```

Comment:
Should this line be outside of the conditional still, so it runs even when !unattended?